### ISSUE TITLE
msgpackr broken with node 12 (still LTS)

### ISSUE THREAD

**Issue by Skywalker13** [kriszyp/msgpackr#37]
Hello,
I must use node 12 and it's broken because it uses new features
```
./node_modules/msgpackr/dist/node.cjs:1004
                let maxSharedStructures = options.maxSharedStructures ?? (hasSharedStructures ? 32 : 0);
                                                                       ^

SyntaxError: Unexpected token '?'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
```

The "end of life" for node 12 is 2022
![image](https://user-images.githubusercontent.com/397862/131329233-d76e8921-34af-4243-8df8-ca941406f063.png)

Thanks  
Regards

**Comment by kriszyp** [comment]
Thanks for the report, published fix in v1.4.3.

**Comment by Skywalker13** [comment]
Thank you very much
